src/dialects/mysql/query-generator.js

```diff
@@ -162,12 +172,12 @@ export class MySqlQueryGenerator extends AbstractQueryGenerator {
     return `SHOW FULL COLUMNS FROM ${table};`;
   }
 
-  showTablesQuery(database) {
-    let query = 'SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = \'BASE TABLE\'';
-    if (database) {
-      query += ` AND TABLE_SCHEMA = ${this.escape(database)}`;
+  showTablesQuery(schemaName) {
+    let query = 'SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = \'BASE TABLE\'';
+    if (schemaName) {
+      query += ` AND TABLE_SCHEMA = ${this.escape(schemaName)}`;
     } else {
-      query += ' AND TABLE_SCHEMA NOT IN (\'MYSQL\', \'INFORMATION_SCHEMA\', \'PERFORMANCE_SCHEMA\', \'SYS\', \'mysql\', \'information_schema\', \'performance_schema\', \'sys\')';
```

Comment:
Why has `SYS` been removed from here? And the lowercase versions? Since we do use those in `schemasToSkip`. I see that that's from the original MariaDB implementation but I'm not sure if that's correct for MySQL

Reply:
Accidental removal, but will need to not include it in MariaDB

MySQL:

![image](https://user-images.githubusercontent.com/1280915/190990282-fbaf3c46-81ab-4f30-a9ec-6cabd5e8985a.png)

MariaDB:

![image](https://user-images.githubusercontent.com/1280915/190990311-8bd2efd9-0550-4930-9e26-55e35f4f67e0.png)

Reply:
Done